synapse/rest/admin/__init__.py

```diff
@@ -342,45 +174,15 @@ def __init__(self, hs):
         return 200, purge_status.asdict()
 
 
-class DeactivateAccountRestServlet(RestServlet):
```

Comment:
again: please don't move the unchanged servlets (at least, not in the same PR). It's much harder to review the actual changes when the diff includes hundreds of lines of moving things around.

Reply:
OK, I will put this change into a separate PR.

Reply:
Done